website/docs/d/cloud_credentials.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+subcategory: "Cloud"
+layout: "aci"
+page_title: "ACI: aci_cloud_credentials"
+sidebar_current: "docs-aci-data-source-aci_cloud_credentials"
+description: |-
+  Data source for Cloud ACI Access Credential to manage the cloud resources
+---
+
+# aci_cloud_credentials #
+
+Data source for Cloud ACI Access Credential to manage the cloud resources
+Note: This resource is supported in Cloud APIC only.
```

Comment:
We probably want this note in the other ones too.
It is a datasource here though